pkg/perf/perf_suite_test.go

```diff
@@ -0,0 +1,203 @@
+package perf_test
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"runtime"
+	"sync"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/resty.v1"
+	. "zotregistry.io/zot/pkg/perf"
+	. "zotregistry.io/zot/pkg/perf/images"
+)
+
+// nolint: gochecknoglobals
+var (
+	username                    string
+	password                    string
+	address                     string
+	repo                        string
```

Comment:
This is a good idea for allowing folks to specify, so we can actually test on a live server if need be.